t/058-tcp-socket.t

```diff
@@ -3298,6 +3299,7 @@ failed to receive a line: closed []
 close: 1 nil
 --- no_error_log
 [error]
+--- skip_eval: 3: system("ping6 -c 1 ::1 >/dev/null 2>&1") ne 0
```

Comment:
@chipitsine Other nginx modules of openresty have test cases against IPv6 as well. So better abstract it away? We can check if `--with-ipv6` is present in the `nginx -V` output instead of calling `ping6`.